File: Programmers/Level1/예산.py
def solution(d, budget):
    answer = 0
    sumOfD = 0
    d.sort()
    for i in range(len(d)):
        sumOfD += d[i]
        if(sumOfD > budget):
            answer = i
            break
        else:
            answer = i+1

    return answer

File: Swift 코딩테스트 연습/예산.swift
import Foundation

func solution(_ d:[Int], _ budget:Int) -> Int {
    var count:Int = 0
    var index:Int = 0
    var leftBudget:Int = budget
    var registerList:[Int] = d
    registerList.sort()

    while index < registerList.count {
        leftBudget -= registerList[index]
        if leftBudget < 0 { break }
        index += 1
        count += 1
    }

    return count
}